boards/common/arduino-atmega/Kconfig

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD_COMMON_ARDUINO_ATMEGA
+    bool
+    select HAS_PERIPH_ADC
+    select HAS_PERIPH_I2C
+    select HAS_PERIPH_PWM
+    select HAS_PERIPH_SPI
+    select HAS_PERIPH_TIMER
+    select HAS_PERIPH_UART
+    select HAS_ARDUINO
+    select HAS_ARDUINO_PWM
```

Comment:
```suggestion
    # Put defined MCU peripherals here (in alphabetical order)
    select HAS_PERIPH_ADC
    select HAS_PERIPH_I2C
    select HAS_PERIPH_PWM
    select HAS_PERIPH_SPI
    select HAS_PERIPH_TIMER
    select HAS_PERIPH_UART
    # Various other features (if any)
    select HAS_ARDUINO
    select HAS_ARDUINO_PWM
```

Reply:
This kind of comments could be added everywhere

Reply:
Ok